shared/chat/conversation/normal/index.native.tsx

```diff
@@ -28,40 +27,34 @@ const Offline = () => (
   </Box>
 )
 
-class Conversation extends React.PureComponent<Props> {
-  render() {
-    return (
-      <Box2 direction="vertical" fullWidth={true} fullHeight={true}>
-        {this.props.threadLoadedOffline && <Offline />}
-        <HeaderArea
-          onToggleInfoPanel={this.props.onToggleInfoPanel}
-          conversationIDKey={this.props.conversationIDKey}
+const Conversation = (props: Props) => {
```

Comment:
if you're going from pure to a functional component we should eval if it still needs to be pure or not, if so use `React.memo`